charmcraft/commands/store/__init__.py

```diff
@@ -87,6 +87,17 @@ class LoginCommand(BaseCommand):
         access to Charmhub at the CLI (if `--export` option was not used
         otherwise it will only save the credentials in the indicated file).
 
+        If `--export <file>` option was used, a secret credentials file will
+        be created. And the file can be used to set "CHARMCRAFT_AUTH" 
+        environment variable.
+
+            e.g. `export CHARMCRAFT_AUTH=$(cat secret)`
```

Comment:
pedantic: These indented lines get automatically formatted as markdown for the forums, so I would remove the `e.g.` and the backticks from this.

For example, you can see this automatic formatting with `charmcraft fetch-lib --help --format=markdown` and how it looks on the forums here: https://juju.is/docs/sdk/charmcraft-fetch-lib